pkg/plugins/github/label.go

```diff
@@ -0,0 +1,231 @@
+package github
+
+import (
+	"context"
+
+	"github.com/shurcooL/githubv4"
+	"github.com/sirupsen/logrus"
+)
+
+// repositoryLabel hold specific label information returned from Github API
+type repositoryLabel struct {
+	ID          string
+	Name        string
+	Description string
+}
+
+// getRepositoryLabelsInformation queries GitHub Api to retrieve every labels configured for a repository
+// then only return those matching label name specified via an updatecli configuration
+func (g *Github) getRepositoryLabelsInformation() ([]repositoryLabel, error) {
+
+	/*
+		https://developer.github.com/v4/explorer/
+
+			query($owner: String!, $name: String!) {
+				rateLimit {
+					cost
+					remaining
+					resetAt
+				}
+				repository(owner: $owner, name: $name){
+					labels (last: 5, before: $before) {
+						totalCount
+						pageInfo {
+							hasNextPage
+							endCursor
+						}
+						edges {
+							node {
+								id
+								name
+								description
+							}
+							cursor
+						}
+					}
+				}
+			}
+
+	*/
```

Comment:
```suggestion
```

Reply:
I like to keep in the comment that snippet that I used in Github console for future iteration